<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainView</class>
    <widget class="QWidget" name="MainView">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1018</width>
                <height>630</height>
            </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
                <widget class="QTableWidget" name="availableFiles">
                    <property name="editTriggers">
                        <set>QAbstractItemView::NoEditTriggers</set>
                    </property>
                    <property name="alternatingRowColors">
                        <bool>true</bool>
                    </property>
                    <property name="selectionMode">
                        <enum>QAbstractItemView::NoSelection</enum>
                    </property>
                    <property name="showGrid">
                        <bool>false</bool>
                    </property>
                    <property name="gridStyle">
                        <enum>Qt::SolidLine</enum>
                    </property>
                    <property name="wordWrap">
                        <bool>false</bool>
                    </property>
                    <property name="columnCount">
                        <number>4</number>
                    </property>
                    <attribute name="verticalHeaderVisible">
                        <bool>false</bool>
                    </attribute>
                    <column>
                        <property name="text">
                            <string>Name</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>Modified date</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>Size</string>
                        </property>
                    </column>
                    <column>
                        <property name="text">
                            <string>Path</string>
                        </property>
                    </column>
                </widget>
            </item>
            <item row="1" column="0" colspan="3" alignment="Qt::AlignBottom">
                <widget class="QPlainTextEdit" name="logData">
                    <property name="minimumSize">
                        <size>
                            <width>0</width>
                            <height>192</height>
                        </size>
                    </property>
                    <property name="font">
                        <font>
                            <family>Monospace</family>
                        </font>
                    </property>
                    <property name="readOnly">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
            <item row="0" column="1" alignment="Qt::AlignRight">
                <widget class="QGroupBox" name="groupBox">
                    <property name="title">
                        <string>Render queue</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_2">
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <property name="spacing">
                            <number>0</number>
                        </property>
                        <item row="0" column="0">
                            <widget class="QListWidget" name="renderQueue">
                                <property name="editTriggers">
                                    <set>QAbstractItemView::NoEditTriggers</set>
                                </property>
                                <property name="selectionMode">
                                    <enum>QAbstractItemView::ExtendedSelection</enum>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
